Assets/GothicVR/Scripts/World/DijkstraWaypoint.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem.LowLevel;
+
+namespace GVR
+{
+    public class DijkstraWaypoint : IComparable<DijkstraWaypoint>
```

Comment:
Please add a few comments. Like
* Why is coveredDistance 0, why summedDistance 999
* You could also think of making this a Struct instead of class, as it contains only fields/properties.

Reply:
added comments
when I change it to struct this is what I get
![image](https://github.com/GothicVRProject/GothicVR/assets/43158992/1e924031-71ef-4471-94f2-eb79ab78ad8f)